pythonforandroid/recipes/argon2-cffi/__init__.py

```diff
@@ -0,0 +1,9 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+
+class Argon2Recipe(CompiledComponentsPythonRecipe):
+    version = '20.1.0'
+    url = 'https://github.com/hynek/argon2-cffi/archive/master.zip'
+    depends = ['setuptools', 'cffi']
+    call_hostpython_via_targetpython = False
+
```

Comment:
flake8 requires an extra blank line here.

Reply:
okay

Reply:
it doesn't like the fact that the "blank line" contains whitespace.